src/server/dfly_main.cc

```diff
@@ -733,9 +552,10 @@ void PrintBasicUsageInfo() {
 
 void ParseFlagsFromEnv() {
   if (getenv("DFLY_PASSWORD")) {
-    LOG(WARNING)
+    LOG(FATAL)
         << "DFLY_PASSWORD environment variable is being deprecated in favour of DFLY_requirepass";
```

Comment:
Remove the word "being"?
(also I'd prefer the American spelling - "in favor of")